lib/interface_quda.cpp

```diff
@@ -8998,8 +8921,10 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
 	t1 = MPI_Wtime();	
```

Comment:
The if(smethod==0) statement should be removed. There's no other case than smethod=0.